dynawo/sources/Models/Modelica/Dynawo/Electrical/Photovoltaics/WECC/PVVoltageSource_INIT.mo

```diff
@@ -30,8 +30,8 @@ model PVVoltageSource_INIT "Initialization model for WECC PV model with a voltag
   parameter Types.ApparentPowerModule SNom "Nominal apparent power in MVA";
   parameter Types.PerUnit RPu "Resistance of equivalent branch connection to the grid in pu (base SnRef)";
   parameter Types.PerUnit XPu "Reactance of equivalent branch connection to the grid in pu (base SnRef)";
-  parameter Types.PerUnit RSourcePu "Source resistance in per unit (typically set to zero, typical: 0..0.01)";
-  parameter Types.PerUnit XSourcePu "Source reactance in per unit (typical: 0.05..0.2)";
+  parameter Types.PerUnit RSourcePu "Source resistance in pu (base SnRef) (typically set to zero, typical: 0..0.01)";
```

Comment:
As it is a resistance, the voltage base should also be given. Same for XSourcePu. Also, have you checked it is SnRef and not SNom ?